app/controllers/devise_token_auth/concerns/resource_finder.rb

```diff
@@ -12,10 +12,9 @@ def get_case_insensitive_field_from_resource_params(field)
     q_value
   end
 
-  def find_resource(field, value)
+  def find_resource(field, value, provider)
```

Comment:
I don't understand why the parameter, this won't call the method

Reply:
Hi, could you help me out with how to go about overriding the concern in my project that utilises this repo? I would write a concern that extends `DeviseTokenAuth::Concerns::ResourceFinder` with a ```def provider``` and override the controllers to `include customResourceFinder`. Is this right?

Reply:
I think that won't work, you need to create a concern with `def provider` in there and add that concern in each override controller. But if you need to add it to the registrations controller, you add it there. But you need to add PR here with only this:
<img width="368" alt="screen shot 2017-10-27 at 12 56 28 pm" src="https://user-images.githubusercontent.com/3474627/32113320-54a07594-bb16-11e7-8dd8-f3c1a4728d52.png">